src/index.md

```diff
@@ -1,7 +1,6 @@
 ---
 layout: index
 title: Welcome to Workbox
-excerpt: Workbox is a collection of JavaScript libraries for Progressive Web Apps
```

Comment:
I've added it directly into the template so it's the default description for every page